src/main/java/com/exasol/sql/dql/select/Select.java

```diff
@@ -79,6 +84,32 @@ public Select function(final FunctionName functionName, final String derivedColu
         return this;
     }
 
+    /**
+     * Add a cast function.
+     *
+     * @param value value to cast
+     * @param type  type to cast the value to
+     * @return <code>this</code> instance for fluent programming
+     */
+    public Select cast(final ValueExpression value, final DataType type) {
```

Comment:
If we add all special cases like this, this class will explode. We need to think about the design

Reply:
Another problem, this class is a `common` SELECT, but you now add a method for an Exasol-specific function here, it's not good